api/loader/fileloader.go

```diff
@@ -313,6 +313,9 @@ func (fl *fileLoader) Load(path string) ([]byte, error) {
 			return nil, err
 		}
 		defer resp.Body.Close()
+		if resp.StatusCode < 200 || resp.StatusCode > 299 {
+			return nil, fmt.Errorf("URL returned error %d (%s)", resp.StatusCode, http.StatusText(resp.StatusCode))
```

Comment:
is it possible to write a test for this in `api/krusty/remote_load_test.go`? Something similar to `TestRemoteResource`, but with a bad URL that results in this error?

Reply:
Done!